src/ResourceManager/Sql/Commands.Sql/Auditing/AuditingHelpMessages.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Microsoft.Azure.Commands.Sql.Auditing
+{
+    public static class AuditingHelpMessages
```

Comment:
Shouldn't these go into the resources file for this dll so they could be localized, if needed?

Reply:
@markcowl 
All messages are hard coded everywhere else (As the arguments must be constants)

Reply:
No, this really needs to be in the resources file, and you can use the constants from there in the same way.

Reply:
@markcowl 
Mark, no one uses resources for help messages, Why should I start using them?
All the help messages used in this attribute are hardcoded in the code (Not just SQL). Are you sure that the localization is not done on the md files?
Besides the generated resources strings are static, and not constants, and cannot be used without editing the attribute itself which will need additional work (And I don't think you want me to do that).

Reply:
@ranisha2 Yes, sorry, you are correct, I  had forgotten these aren't represented as static strings.  Apologies for the bad advice.